api/location/schema.py

```diff
@@ -83,8 +86,7 @@ def mutate(self, info, location_id, **kwargs):
             validate_country_field(**kwargs)
         if "image_url" in kwargs:
             validate_url(**kwargs)
-        if "abbreviation" in kwargs:  # noqa
```

Comment:
Hello @TeamoreA,   kindly throw more light on why this condition was removed

Reply:
@nakatuddesuzan  It was only checking empty fields on abbreviation and since I wanted also to validate structure field, it would have been repetition for me to add another if-condition for the structure field.